### ISSUE TITLE
Issue with Azure function v4 alpha9

### ISSUE THREAD

**Issue by gartu** [apollo-server-integrations/apollo-server-integration-azure-functions#42]
Hello!
The latest update of Azure Function v4 alpha9 has caused some major changes, and unfortunately, the project is no longer working. Some interfaces have been modified and even the function parameter order has been shuffled around.

I've pulled a version locally and made the necessary changes for my personal project. It's not a big deal, but if you're interested, I can push my branch to you.

Thanks for all the work in developing this module!
Robin

**Comment by nnoce14** [comment]
Hi, would you be able to create a pull request or publish a fork for your branch? I am trying to use this package and I'm encountering the same problem as you with the latest @azure/functions. I would like to see what changes you made to adapt the package to adhere to Azure Functions v4.

**Comment by aaronpowell** [comment]
Yes, the v4 of Azure Functions is currently not supported as there's a lot of changes to the underlying data structures and types that are incompatible with Azure Functions v3.

While Azure Functions v4 is in public preview I'm going to hold off having it supported OOTB.
